realm/realm-library/src/main/cpp/io_realm_internal_Util.cpp

```diff
@@ -53,6 +53,10 @@ JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM* vm, void*)
         java_lang_double_init = env->GetMethodID(java_lang_double, "<init>", "(D)V");
         java_util_date = GetClass(env, "java/util/Date");
         java_util_date_init = env->GetMethodID(java_util_date, "<init>", "(J)V");
+#if REALM_ENABLE_SYNC
+        java_syncmanager_class = GetClass(env, "io/realm/SyncManager");
```

Comment:
duplicate we're already loading "io/realm/SyncManager"  via`java_syncmanager` in `JNI_OnLoad`